src/FerriteGmsh.jl

```diff
@@ -112,27 +118,27 @@ function toboundary(dim::Int)
     return boundarydict
 end
 
-function _add_to_facesettuple!(facesettuple::Set{FaceIndex}, boundaryface::Tuple, faces)
-    for (eleidx, elefaces) in enumerate(faces)
+function _add_to_facetsettuple!(facesettuple::Set{FacetIndex}, boundaryface::Tuple, facets)
```

Comment:
The local scope clears that there is no conflict with the `const facets` definition, however, it would be somewhat nicer to rename it, since in the function below `facets` is called from the higher scope. This was already an issue before the PR (with Ferrite.faces), but if you feel the same, then we can change it.

Reply:
Ah, missed changing that here (did it below in `tofacetsets`). Coming up!